lib/unit/test/unit/Cardano/Wallet/Api/Malformed.hs

```diff
@@ -215,20 +215,24 @@ instance Wellformed (PathParam ApiDRepSpecifier) where
     wellformed = PathParam <$>
         [ "abstain"
         , "no_confidence"
-        , "drep15k6929drl7xt0spvudgcxndryn4kmlzpk4meed0xhqe25nle07s"
-        , "drep_script1hwj9yuvzxc623w5lmwvp44md7qkdywz2fcd583qmyu62jvjnz69"
+        , "drep1ytje8qacj9dyua6esh86rdjqpdactf8wph05gdd72u46axcvy952y" --28-byte payload with key hash byte prefix ('22' in hex prefixed)
+        , "drep1y0je8qacj9dyua6esh86rdjqpdactf8wph05gdd72u46axcvk492r" --28-byte payload with script hash byte prefix ('23' in hex prefixed)
         ]
 
 instance Malformed (PathParam ApiDRepSpecifier) where
     malformed = first PathParam <$>
-        [ (T.replicate 64 "ś", msg)
-        , (T.replicate 63 "1", msg)
-        , (T.replicate 65 "1", msg)
-        , ("something", msg)
-        , ("no-confidence", msg)
+        [ (T.replicate 64 "ś", msg1)
+        , (T.replicate 63 "1", msg1)
+        , (T.replicate 65 "1", msg1)
+        , ("something", msg1)
+        , ("no-confidence", msg1)
+        , ("drep15k6929drl7xt0spvudgcxndryn4kmlzpk4meed0xhqe25nle07s",msg2) --28-byte payload without byte prefix correct hrp prefix
```

Comment:
Suggestion: use a descriptive name instead of the hard coded value + comment

```
    , (payloadWithoutCorrectBytePrefix, msg2)
  where
       payloadWithoutCorrectBytePrefix = "drep15k6929drl7xt0spvudgcxndryn4kmlzpk4meed0xhqe25nle07s"   
 
```

=> no need for a comment

Reply:
good idea, done